Wikipedia/Code/ArticleViewController+Editing.swift

```diff
@@ -30,6 +31,7 @@ extension ArticleViewController {
         } else {
             present(navigationController, animated: true)
         }
+        EditAttemptFunnel.shared.logInit(articleURL: articleURL)
```

Comment:
Apologies if I'm missing something, but do we need to call this in both `showEditorForSectionOrTitleDescription` and here in `showEditorForSection`? Would it suffice to just have the call in `showEditorForSectionOrTitleDescription`?

Reply:
You're right, it would duplicate the event, good catch!